src/io/hdf5.hpp

```diff
@@ -0,0 +1,52 @@
+/*
+  This file is part of ESPResSo++.
```

Comment:
years missing

Reply:
The header is very inconsistent throughout `ESPResSo++`. How do you want it to be?